schema audit: show field/tag IDs and usage locations

## Enhancement

`supertag schema audit` findings should include node IDs for fields and tags, along with where they are used, to make issues actionable.

### Current Output

```
❌ [ERROR] Field "To" has conflicting types: options, text
  → Standardize "To" to a single type (used as: Gift:options, email:text, flight:text)
```

### Desired Output

Something like:

```
❌ [ERROR] Field "To" has conflicting types: options, text
  → Gift#abc123 → To (def456): options
  → email#ghi789 → To (jkl012): text
  → flight#mno345 → To (pqr678): text
  → Standardize "To" to a single type
```

## Rationale

Without IDs, the user has to manually search for the conflicting fields to resolve them. With IDs, they can directly navigate to the relevant tags/fields in Tana or use other `supertag` commands to inspect them.

This applies to all audit findings — orphan tags, unused fields, conflicting types, etc.

--- a/src/commands/schema.ts
+++ b/src/commands/schema.ts
@@ -579,6 +579,15 @@ function printTableReport(report: import('../types/schema-audit').SchemaAuditRep
   for (const finding of findings) {
     const icon = severityIcon[finding.severity] || '•';
     console.log(`${icon} [${finding.severity.toUpperCase()}] ${finding.message}`);
+    if (finding.details.usageLocations && finding.details.usageLocations.length > 0) {
+      for (const loc of finding.details.usageLocations) {
+        const fieldPart = loc.fieldId
+          ? ` → ${loc.fieldName || '?'} (${loc.fieldId})`
+          : '';
+        const typePart = loc.dataType ? `: ${loc.dataType}` : '';
+        console.log(`  → ${loc.tagName}#${loc.tagId}${fieldPart}${typePart}`);
+      }
+    }
     if (finding.details.suggestion) {
       console.log(`  → ${finding.details.suggestion}`);
     }
@@ -610,6 +619,16 @@ function printMarkdownReport(report: import('../types/schema-audit').SchemaAudit
   for (const finding of findings) {
     console.log(`### ${finding.severity.toUpperCase()}: ${finding.message}`);
     console.log(`- Detector: ${finding.detector}`);
+    if (finding.details.usageLocations && finding.details.usageLocations.length > 0) {
+      console.log('- Locations:');
+      for (const loc of finding.details.usageLocations) {
+        const fieldPart = loc.fieldId
+          ? ` → ${loc.fieldName || '?'} (\`${loc.fieldId}\`)`
+          : '';
+        const typePart = loc.dataType ? `: ${loc.dataType}` : '';
+        console.log(`  - \`${loc.tagName}\`#\`${loc.tagId}\`${fieldPart}${typePart}`);
+      }
+    }
     if (finding.details.suggestion) {
       console.log(`- Suggestion: ${finding.details.suggestion}`);
     }

--- a/src/services/schema-audit-detectors.ts
+++ b/src/services/schema-audit-detectors.ts
@@ -8,6 +8,7 @@
 import type {
   SchemaDetector,
   SchemaFinding,
+  UsageLocation,
   WorkspaceSchema,
   InheritanceRelation,
 } from '../types/schema-audit';
@@ -81,6 +82,7 @@ export const orphanTagsDetector: SchemaDetector = {
           tagName: s.name,
           instanceCount: 0,
           suggestion: `Consider removing "${s.name}" if no longer needed`,
+          usageLocations: [{ tagId: s.id, tagName: s.name }],
         },
       }));
   },
@@ -103,6 +105,7 @@ export const lowUsageTagsDetector: SchemaDetector = {
           tagName: s.name,
           instanceCount: s.instanceCount,
           suggestion: `Consider merging "${s.name}" into a more general supertag`,
+          usageLocations: [{ tagId: s.id, tagName: s.name }],
         },
       }));
   },
@@ -149,6 +152,17 @@ export const duplicateFieldsDetector: SchemaDetector = {
           .map(id => fields.find(f => f.tagId === id)?.tagName || id)
           .join(', ');
 
+        const usageLocations: UsageLocation[] = unrelatedTags.map(tagId => {
+          const field = fields.find(f => f.tagId === tagId);
+          return {
+            tagId,
+            tagName: field?.tagName || tagId,
+            fieldId: field?.fieldLabelId,
+            fieldName,
+            dataType: field?.inferredDataType || undefined,
+          };
+        });
+
         findings.push({
           detector: 'duplicate-fields',
           severity: 'warning',
@@ -157,6 +171,7 @@ export const duplicateFieldsDetector: SchemaDetector = {
             fieldName,
             relatedIds: unrelatedTags,
             suggestion: `Consider creating a shared parent supertag with "${fieldName}" field`,
+            usageLocations,
           },
         });
       }
@@ -192,6 +207,14 @@ export const typeMismatchDetector: SchemaDetector = {
         .map(f => `${f.tagName}:${f.inferredDataType}`)
         .join(', ');
 
+      const usageLocations: UsageLocation[] = fields.map(f => ({
+        tagId: f.tagId,
+        tagName: f.tagName,
+        fieldId: f.fieldLabelId,
+        fieldName: f.fieldName,
+        dataType: f.inferredDataType || undefined,
+      }));
+
       findings.push({
         detector: 'type-mismatch',
         severity: 'error',
@@ -200,6 +223,7 @@ export const typeMismatchDetector: SchemaDetector = {
           fieldName,
           relatedIds: fields.map(f => f.tagId),
           suggestion: `Standardize "${fieldName}" to a single type (used as: ${examples})`,
+          usageLocations,
         },
       });
     }
@@ -243,6 +267,12 @@ export const unusedFieldsDetector: SchemaDetector = {
             tagName: field.tagName,
             fillRate: 0,
             suggestion: `Consider removing "${field.fieldName}" from "${field.tagName}" if not needed`,
+            usageLocations: [{
+              tagId: field.tagId,
+              tagName: field.tagName,
+              fieldId: field.fieldLabelId,
+              fieldName: field.fieldName,
+            }],
           },
         });
       }
@@ -272,8 +302,15 @@ export const fillRateDetector: SchemaDetector = {
             fieldName: fv.fieldName,
             tagId: fv.tagId,
             tagName: field?.tagName,
+            fieldId: field?.fieldLabelId,
             fillRate: fv.fillRate,
             suggestion: `Review if "${fv.fieldName}" is still useful (${fv.populatedCount}/${fv.totalInstances} populated)`,
+            usageLocations: [{
+              tagId: fv.tagId,
+              tagName: field?.tagName || fv.tagId,
+              fieldId: field?.fieldLabelId,
+              fieldName: fv.fieldName,
+            }],
           },
         };
       });
@@ -333,6 +370,10 @@ export const missingInheritanceDetector: SchemaDetector = {
             details: {
               relatedIds: [tagA, tagB],
               suggestion: `Consider creating a parent supertag with shared fields: ${shared.join(', ')}`,
+              usageLocations: [
+                { tagId: tagA, tagName: tagNameA },
+                { tagId: tagB, tagName: tagNameB },
+              ],
             },
           });
         }

--- a/src/types/schema-audit.ts
+++ b/src/types/schema-audit.ts
@@ -8,6 +8,15 @@
 /** Severity levels for schema findings */
 export type SchemaFindingSeverity = 'error' | 'warning' | 'info';
 
+/** A usage location showing where a tag/field is used */
+export interface UsageLocation {
+  tagId: string;
+  tagName: string;
+  fieldId?: string;
+  fieldName?: string;
+  dataType?: string;
+}
+
 /** A single finding from a schema detector */
 export interface SchemaFinding {
   detector: string;
@@ -22,6 +31,7 @@ export interface SchemaFinding {
     relatedIds?: string[];
     fillRate?: number;
     instanceCount?: number;
+    usageLocations?: UsageLocation[];
   };
   tanaPaste?: string;
 }

